src/native/libs/System.Globalization.Native/pal_locale.m

```diff
@@ -61,7 +60,7 @@ static void GetParent(const char* localeID, char* parent, int32_t parentCapacity
     int32_t i;
 
     if (localeID == NULL)
-        localeID = [NSLocale systemLocale].localeIdentifier.UTF8String;
+        localeID = [NSLocale currentLocale].localeIdentifier.UTF8String;
```

Comment:
this seems unrelated to the system ICU change?

Reply:
yes, will revert it back.